salt/modules/nxos.py

```diff
@@ -382,11 +380,6 @@ def show(commands, raw_text=True, **kwargs):
         salt '*' nxos.show 'show bgp sessions ; show processes' raw_text=False
         salt 'regular-minion' nxos.show 'show interfaces' host=sw01.example.com username=test password=test
     """
-    warn_until(
```

Comment:
Why are we removing the deprecation warnings without removing the code that was scheduled to be removed?

Reply:
Having a conversation with @dmurphy18 about what to do with this..

Reply:
I'll have to work with David on this one... but the tests will need to be updated and this his territory of stuff :) so if we can reasonably get that done, we'll try. if not, we'll have to punt the deprecation warning to the next release